app/[localeCode]/(story)/[slug]/page.tsx

```diff
@@ -16,12 +16,18 @@ interface Props {
 }
 
 async function resolve(params: Props['params']) {
-    const { slug } = params;
+    const { localeCode, slug } = params;
 
     const story = await app().story({ slug });
     if (!story) notFound();
 
-    return { story };
+    const { stories: relatedStories } = await app().stories({
+        limit: 3,
+        locale: localeCode,
+        query: JSON.stringify({ slug: { $ne: slug } }),
```

Comment:
Perhaps specify a sort explicitly? I imagine it's chronological by default, so not needed necessarily.

Reply:
The requirement was to do it as on homepage, and there we do not apply sorting:
https://github.com/prezly/theme-nextjs-bea/blob/main/modules/Stories/Stories.tsx